<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TkmIsp</class>
 <widget class="QWidget" name="TkmIsp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="comlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>COM口</string>
   </property>
  </widget>
  <widget class="QLabel" name="msu0label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MSU0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="msu0filepath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="refresh">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QComboBox" name="com">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="colon">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="msu0openfile">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="msu1openfile">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="colon_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="msu1label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MSU1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="msu1filepath">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="msu0load">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="msu1load">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="tips">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="readchipmsg">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>读芯片信息</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkwrite">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>完成后校验</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="erasechip">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>擦除全部</string>
   </property>
  </widget>
  <widget class="QPushButton" name="msu0readchip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>读取MSU0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writechip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>烧录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearlog">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="msu1readchip">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>读取MSU1</string>
   </property>
  </widget>
  <widget class="QLabel" name="colon_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="netctl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>启动远程调用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="erasedata">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>擦除数据</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
